### PR TITLE
Adding support for the shawkesPad; a modular macro pad.

### PR DESCRIPTION
**Pull request by glassworksSean**
## Description

Adding qmk support for the shawkesPad; a 3D printed, modular macro pad.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/shawkesPad/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# shawkesPad
+
+![shawkesPad]
+
+A short description of the keyboard/project
```
Comment (fauxpark) [inline keyboards/shawkesPad/readme.md:5]: Please fill these out.

File: keyboards/shawkesPad/rules.mk [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# MCU name
+MCU = atmega32u4
+
+BOOTLOADER = atmel-dfu
```
Comment (fauxpark) [inline keyboards/shawkesPad/rules.mk:4]:
```suggestion
# Bootloader selection
BOOTLOADER = atmel-dfu
```

File: keyboards/shawkesPad/rules.mk [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = yes       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE = yes		# Enable encorder support
```
Comment (fauxpark) [inline keyboards/shawkesPad/rules.mk:18]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
ENCODER_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]
Please rename the folder and filenames, and their references, to lowercase.

File: keyboards/shawkesPad/readme.md (hunk 1 above)
Comment (glassworksSean) [reply keyboards/shawkesPad/readme.md]: Resolved via 8fed2ca

File: keyboards/shawkespad/readme.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# shawkesPad
+
+![shawkesPad](https://cdn.discordapp.com/attachments/742152840843362324/742152968723365928/IMG_20200808_213856.jpg)
+===
```
Comment (fauxpark) [inline keyboards/shawkespad/readme.md:4]:
```suggestion
```

File: keyboards/shawkespad/readme.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+A custom, 3D printed, modularly built macro pad.
+
+Keyboard Maintainer: shawkes
+Hardware Supported: Elite-C
+Hardware Availability: [Keebio](https://keeb.io/products/elite-c-low-profile-version-usb-c-pro-micro-replacement-atmega32u4)
```
Comment (fauxpark) [inline keyboards/shawkespad/readme.md:10]:
```suggestion
* Keyboard Maintainer: shawkes
* Hardware Supported: Elite-C
* Hardware Availability: [Keebio](https://keeb.io/products/elite-c-low-profile-version-usb-c-pro-micro-replacement-atmega32u4)
```

File: keyboards/shawkespad/readme.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+Make example for this keyboard (after setting up your build environment):
+
+    make shawkesPad:default
```
Comment (fauxpark) [inline keyboards/shawkespad/readme.md:14]:
```suggestion
    make shawkespad:default
```

File: keyboards/shawkespad/readme.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
```
Comment (fauxpark) [inline keyboards/shawkespad/readme.md:17]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by noroadsleft** [comment]
If this is a handwired board, it should go in `keyboards/handwired/`.

**Pull request closed without merging**